### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by gracieflores**
Replacing Travis with GitHub Actions.

### CODE REVIEW

**Comment by gracieflores** [comment]
@ldko @somexpert This is ready for review.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Requirements
 ------------
 
 * Django 2.2
-* Python == 3.6, 3.7, 3.8
+* Python == 3.6, 3.7, 3.8, 3.9
```
Comment (ldko) [inline README.md]: I would change this to "Python 3.6-3.9", but I don't think it is a big deal :).

**Review by ldko — APPROVED** [review]
Looks like it is working. You should add your name to the Contributors on the README :).

**Review by somexpert — APPROVED** [review]
Looks good to me! :thumbsup:

**Pull request merged**